cmd/utils/flags.go

```diff
@@ -1652,8 +1668,14 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 	if ctx.GlobalIsSet(TriesInMemoryFlag.Name) {
 		cfg.TriesInMemory = ctx.GlobalUint64(TriesInMemoryFlag.Name)
 	}
-	if ctx.GlobalIsSet(AllowInsecureNoTriesFlag.Name) {
-		cfg.NoTries = ctx.GlobalBool(AllowInsecureNoTriesFlag.Name)
+	if ctx.GlobalIsSet(TriesVerifyModeFlag.Name) {
+		cfg.TriesVerifyMode = *GlobalTextMarshaler(ctx, TriesVerifyModeFlag.Name).(*core.VerifyMode)
+		if cfg.TriesVerifyMode == core.FullVerify || cfg.TriesVerifyMode == core.LightVerify {
```

Comment:
Better add comments here, why we should enable trust protocol, and why we should disable diff protocol below.

Reply:
done

Reply:
Need blank between // and the first word, please modify the related comments.